### ISSUE TITLE
`mlflow db upgrade $DB_URI` might cause an issue

### ISSUE THREAD

**Issue by daisukelab** [ymym3412/mlflow-docker-compose#4]
As discussed in following thread, `mlflow db upgrade $DB_URI` could result in an error.

https://github.com/mlflow/mlflow/issues/2444

It happened when I was trying to use this repo, but I couldn't get the container mlflow up, it died actually when trying the command in start.sh.

I commented out to skip the line, then it seems to auto-upgrade db as follows:

```
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 451aebb31d03, add metric step
INFO  [alembic.runtime.migration] Running upgrade 451aebb31d03 -> 90e64c465722, migrate user column to tags
  :
```

Anyway thank you and I'm appreciate sharing nice docker compose file.

**Comment by pomcho555** [comment]
I solved with commenting out `mlflow db upgrade $DB_URI` as first run same as you.

That might relate to this [issue](https://github.com/mlflow/mlflow/issues/2444)

**Comment by daisukelab** [comment]
Thanks for info, I didn't confirm mlflow's fix https://github.com/mlflow/mlflow/issues/2444 , it should resolve this also.
I close now, please somebody re-open if you still see this issue.

**Comment by daisukelab** [comment]
Thanks for info, I didn't confirm mlflow's fix mlflow/mlflow#2444 , it should resolve this also.
I close now, please somebody re-open if you still see this issue.
